utils/hooks.js

```diff
@@ -14,3 +15,11 @@ export const usePageContext = (previewContextObject) => {
 export const usePendingChanges = () => {
 	return useContext(PendingChanges);
 };
+
+export const useCallbackThrottled = (fn, timeout, throttleOptions, deps = []) => 
+	useCallback(throttle(fn, timeout, throttleOptions), deps);
+
+export const useEffectThrottled = (fn, timeout, throttleOptions, args = []) => {
+	const execute = useCallbackThrottled(fn, 1000, throttleOptions);
```

Comment:
My guess is that you meant to eventually forward `timeout` instead of `1000` into `useCallbackThrottled`?

Reply:
Yep good catch!